test/sol6/networkHelper.js

```diff
@@ -78,7 +78,7 @@ async function setupReserves
             'address': reserve.address,
             'instance': reserve,
             'reserveId': reserveId,
-            'onChainType': ReserveType.FPR,
+            'onChainType': ReserveType.CUSTOM,
```

Comment:
good idea.
mock = custom

but it might affect fee entitled data in many places. right?

Reply:
Should be fine with the configuration set.